opentelemetry-otlp/allowed-external-types.toml

```diff
@@ -16,6 +16,9 @@ allowed_external_types = [
     "tonic::status::Code",
     "tonic::status::Status",
     "tonic::metadata::map::MetadataMap",
+    "tonic::transport::channel::tls::ClientTlsConfig",
+    "tonic::transport::tls::Certificate",
+    "tonic::transport::tls::Identity",
     "tonic::transport::channel::Channel",
     "tonic::transport::error::Error",
     "tonic::service::interceptor::Interceptor",
```

Comment:
I assume that all of these should be reexported and removed?

Reply:
Hi @bantonsson, thanks for the review!
The added types are the ones now re-exported.
I also updated the CI script to check allowed external types with all features enabled, because the previous configuration was correct but wasn't actually enforced (some re-exported types are feature-gated).
I've also cleaned up the old types that are no longer re-exported.

Reply:
Thanks for the explanation @MathieuTricoire. Didn't know how that plugin works.